website/docs/cli/configuration/configuration.mdx

```diff
@@ -24,15 +24,58 @@ Think of this file as where you [bootstrap the settings or configuration of your
+
+## Configuration File (`atmos.yaml`)
+
+The Atmos (CLI) configuration is loaded from the following locations, in order of precedence, and deep-merged: configurations listed earlier have higher priority, meaning they override any conflicting sections defined in subsequent locations:
+- The `--config` parameter is the relative or absolute path to a valid configuration file or directory containing strictly atmos configuration file(s). It has the highest precedence, meaning it supersedes everything.
+- The `$XDG_CONFIG_HOME/atmos/atmos.yaml` file (if `$XDG_CONFIG_HOME` is set) allows the user to specify their preferences and overrides. It is only superseded by the `--config` flag.
+- The environment variable `ATMOS_CLI_CONFIG_PATH` should point to a folder (or a specific file). This is mutually exclusive with the `--config` parameter. If `--config` is specified, then the `ATMOS_CLI_CONFIG_PATH` is ignored. Atmos will only search for the configuration file in the specified folder. Note that `.yaml` and `.yml` file extensions are treated interchangeably and are loaded in lexicographical order. Atmos will search for the following files, in order, relative to the `ATMOS_CLI_CONFIG_PATH` environment variable.
+  From the current directory:
+  - `atmos.yaml`
+  - `.atmos.yaml`
+  - `.github/atmos.yaml`
+  - `atmos.d/**/*`
+  - `.atmos.d/**/*`
 - Home directory (`~/.atmos/atmos.yaml`)
-- Current directory (`./atmos.yaml`)
-- Environment variable `ATMOS_CLI_CONFIG_PATH` (the ENV var should point to a folder without specifying the file name)
+- The `atmos.d/` directory is loaded relative to the `base_path`. For an example, see [`examples/demo-atmos.d`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos.d)
+- System directory (`/usr/local/etc/atmos/atmos.yaml` on Linux, `%LOCALAPPDATA%/atmos/atmos.yaml` on Windows). This has the lowest precedence of all configuration files.
+
+Additionally, Atmos supports `imports` of other CLI configurations. Use imports to break large Atmos CLI configurations into smaller ones, such as organized by top-level section. File imports are relative to the base path (if `import`  section is set in the config). All imports are processed and deep-merged in order, so that the last file in the list supersedes settings in the preceding imports. For an example, see [`examples/demo-atmos-cli-imports`](https://github.com/cloudposse/atmos/tree/main/examples/demo-atmos-cli-imports).
+
+Imports can be any of the following:
+- Remote URL (if `import` section is set in the config)
+- Specific Path (if `import` section is set in the config)
+- Wildcard globs (`*`), including recursive globs (`**`), can be combined (e.g., `**/*` matches all files and subfolders recursively). Only files ending in `.yml` or `.yaml` will be considered for import when using globs.
+
+For example, we can import from multiple locations like this:
+
+```yaml
+import:
```

Comment:
We do not intend to support templated imports at this time. Also, this syntax might change based on viper?